frontend/talentsearch/src/js/components/Home/partials/Opportunities/Opportunities.tsx

```diff
@@ -0,0 +1,173 @@
+          data-h2-height="base(auto) l-tablet(40%)"
+          data-h2-width="base(250%) l-tablet(auto)"
+          data-h2-max-width="base(initial)"
+          src={imageUrl(TALENTSEARCH_APP_DIR, "Desktop_Graphics_dark_1.png")}
+          alt=""
+        />
+        <div
+          data-h2-background-color="base(tm-linear-divider)"
+          data-h2-offset="base(0, 0, auto, 0)"
+          data-h2-display="base(block)"
+          data-h2-height="base(x1)"
+          data-h2-position="base(absolute)"
+        />
+      </div>
+      <div
+        data-h2-position="base(relative)"
+        data-h2-container="base(center, large, x1) p-tablet(center, large, x2)"
+      >
+        <div data-h2-padding="base(x3, 0) p-tablet(x5, 0, x4, 0) l-tablet(x7, 0, x6, 0)">
+          {/* Requires an ID and a translated text string */}
+          <Heading
+            level="h2"
+            Icon={MagnifyingGlassCircleIcon}
+            color="yellow"
+            data-h2-margin="base(0, 0, x0.5, 0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Ongoing recruitment",
+              id: "nEMeaQ",
+              description: "Heading for the recruitment opportunities",
+            })}
+          </Heading>
+          <div
+            data-h2-display="base(grid)"
+            data-h2-grid-template-columns="base(1fr) p-tablet(repeat(2, minmax(0, 1fr))) l-tablet(repeat(3, minmax(0, 1fr)))"
+            data-h2-gap="base(x2) p-tablet(x3)"
+            data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)"
+          >
+            <Block
+              content={{
+                color: "yellow",
+                title: intl.formatMessage({
+                  defaultMessage: "Jobs in digital government",
+                  id: "+cBKDC",
+                  description:
+                    "Heading for the digital government job opportunities",
+                }),
+                summary: intl.formatMessage({
+                  defaultMessage:
+                    "Check out the latest GC opportunities in digital and tech, from entry level to management. Find a team, make a difference, and be inspired.",
+                  id: "951Oju",
+                  description:
+                    "Description for the digital government job opportunities",
+                }),
+                link: {
+                  path: diPaths.allPools(),
+                  label: intl.formatMessage({
+                    defaultMessage: "Browse IT jobs",
+                    id: "zNvXSs",
+                    description:
+                      "Link text for IT jobs in government call to action",
+                  }),
+                },
+              }}
+            />
+            <Block
+              content={{
+                color: "blue",
+                title: intl.formatMessage({
+                  defaultMessage: "Indigenous Apprenticeship Program",
```

Comment:
That is a good question. My assumption is that is can be consider a proper noun since it is the name of a program which is why it is capitalized in such a way?

Am I correct on this one @JoshBeveridge ?

Reply:
@esizer has it right - [All titles should be sentence case, excluding proper nouns.](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-1-1)